server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/AuthenticationRoutes.scala

```diff
@@ -0,0 +1,53 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.http
+
+import java.util.stream.Stream
+
+import org.apache.james.jmap.JMAPRoute.Action
+import org.apache.james.jmap.JMAPUrls.AUTHENTICATION
+import org.apache.james.jmap._
+import org.slf4j.{Logger, LoggerFactory}
+
+object AuthenticationRoutes {
+  private val LOGGER = LoggerFactory.getLogger(AuthenticationRoutes.getClass)
+}
+
+class AuthenticationRoutes extends JMAPRoutes {
```

Comment:
RFC-8621 don't define authentication, we decided to go along with Simple authentication & JWT

Hence there is no need for Authentication routes ;-)